src/index.html

```diff
@@ -17,6 +17,101 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#logo"
+        class="logo_container"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="moyo"
+          class="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="container">
+          <li class="nav_list">
```

Comment:
```suggestion
        <ul class="nav_list">
          <li class="nav_litem">
```
`ul` means an unordered list
`li` means list item

Reply:
fixed